app/src/main/java/com/sopt/dive/feature/login/LoginScreen.kt

```diff
@@ -51,14 +51,13 @@ import com.sopt.dive.core.ui.noRippleClickable
 fun LoginScreen(
     modifier: Modifier = Modifier,
     onLoginClick: (String, String) -> Unit,
-    onSignUpClick: () -> Unit
+    onSignUpClick: () -> Unit,
+    viewModel: LoginViewModel = androidx.lifecycle.viewmodel.compose.viewModel(
+        factory = LoginViewModelFactory(LocalContext.current)
+    )
 ) {
     val context = LocalContext.current
-    val userPreferences = UserPreferences.getInstance(context)
-
-    var idText by remember { mutableStateOf("") }
-    var pwText by remember { mutableStateOf("") }
-
+    val uiState by viewModel.uiState.collectAsState()
```

Comment:
```suggestion
    val uiState by viewModel.uiState.collectAsStateWithLifecycle()
```
로 써보면 좋습니다~ 이유는 저번주 코멘트로도 남겼으니까 이번주엔 넘어갈게요